pymc3/distributions/continuous.py

```diff
@@ -1654,6 +1655,93 @@ def logcdf(self, value):
         )
 
 
+class AsymmetricLaplace(Continuous):
+    r"""
+    Asymmetric-Laplace log-likelihood.
+
+    The pdf of this distribution is
+
+    ..math::
+        {f(x|\\b,\kappa) =
+            \left({\frac{\\b}{\kappa + 1/\kappa}}\right)\,e^{-(x)\\b\,s\kappa ^{s}}}
+        where s = sgn(x)
+
+    Parameters
+    ----------
+    b:
+        Scale parameter (b > 0)
+    kappa:
+        Symmetry parameter (kappa > 0)
```

Comment:
I sense `b` and `kappa` are subjected to `float` values.
```suggestion
    b: float
        Scale parameter (b > 0)
    kappa: float
        Symmetry parameter (kappa > 0)
```